recipe/meta.yaml

```diff
@@ -1,29 +1,28 @@
 {% set name = "radical.pilot" %}
-{% set version = "1.49.2" %}
-{% set sha256 = "f49f40426df2326b1c724b4a4bcd34c2fe07b60c96040e55c0a129a3d13646cc" %}
+{% set version = "1.52.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radical_pilot-{{ version }}.tar.gz
+  sha256: ad477f68032e8a52b7c78bfa69557ebb8aba3ed69347590ae77620fe59114052
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-  skip: true  # [py2k or win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
```

Comment:
I don't see any reason this package can't be `noarch: python` but please let me know if I just missed it.